source/configuration.txt

```diff
@@ -249,15 +249,40 @@ Partitioner Configuration
        **Default:** ``_id``
 
    * - ``partitionSizeMB``
-     - The size (in MB) for each partition
+     - The size (in MB) for each partition. Smaller partition sizes 
+       create more partitions containing fewer documents.
 
        **Default:** ``64``
 
    * - ``samplesPerPartition``
-     - The number of sample documents to take for each partition.
+     - The number of sample documents to take for each partition in 
+       order to establish a ``partitionKey`` range for each partition. 
+       A greater number of ``samplesPerPartition`` increases the 
+       precision of finding partition ranges that match the 
+       ``partitionSizeMB`` you specify.
+       
+       .. note::
+       
+          For sampling to improve performance, ``samplesPerPartition`` 
+          should be fewer than the number of documents within each of 
```

Comment:
Question: Should "should" be "must" here?

```suggestion
          must be fewer than the number of documents within each of 
```

Reply:
Yes, good catch - 'must' in order to improve performance